tests/bud.bats

```diff
@@ -4238,13 +4238,13 @@ _EOF
   run_buildah tag image-amd localhost/${SAFEIMAGE_NAME}:${SAFEIMAGE_TAG}
   run_buildah build --pull=false -q --arch=arm64 -t image-arm $WITH_POLICY_JSON ${mytmpdir}
```

Comment:
not sure if this also needs to be changed to aarch64. I'm hoping gating will point that out?